infrastructure/templates/template.json

```diff
@@ -972,7 +831,84 @@
       {
         "name": "[concat(parameters('subscription'),'PubStatsDBBlockedByFirewall')]",
         "description": "Public Statistics DB Blocked By Firewall",
-        "resourceId": "[resourceId('Microsoft.Sql/servers/databases', variables('publicSqlServerName'), 'public-statistics')]",
+        "resourceId": "[resourceId('Microsoft.Sql/servers/databases', variables('publicSqlServerName'), variables('publicStatisticsDbName'))]",
+        "resourceType": "microsoft.sql/servers/databases",
+        "metricName": "blocked_by_firewall",
+        "threshold": 0,
+        "operator": "GreaterThan",
+        "timeAggregation": "Total",
+        "criterionType": "StaticThresholdCriterion"
+      },
+      {
+        "name": "[concat(parameters('subscription'),'StatisticsReplicaDBCpuPercent')]",
+        "description": "Statistics Replica DB CPU percentage",
+        "resourceId": "[resourceId('Microsoft.Sql/servers/databases', variables('publicSqlServerName'), variables('statisticsReplicaDbName'))]",
+        "resourceType": "microsoft.sql/servers/databases",
+        "metricName": "cpu_percent",
+        "threshold": 85,
+        "operator": "GreaterThan",
+        "timeAggregation": "Average",
+        "criterionType": "StaticThresholdCriterion"
+      },
+      {
+        "name": "[concat(parameters('subscription'),'StatisticsReplicaDBDataIOPercent')]",
+        "description": "Statistics Replica DB Data IO Percentage",
+        "resourceId": "[resourceId('Microsoft.Sql/servers/databases', variables('publicSqlServerName'), variables('statisticsReplicaDbName'))]",
+        "resourceType": "microsoft.sql/servers/databases",
+        "metricName": "physical_data_read_percent",
+        "threshold": 50,
```

Comment:
Threshold seems a bit low? Maybe we should review these where they are set to 50 elsewhere too.

Reply:
Yeah fine by me - currently in this PR I am just lifting and shifting.

Reply:
Cool, I might raise this next time we get one of the threshold alerts, rather than creating a ticket now.